src/main/scala/com/schuwalow/todo/repository/TodoRepository.scala

```diff
@@ -9,29 +9,23 @@ import com.schuwalow.todo.{
   TodoItemPostForm
 }
 
-trait TodoRepository extends Serializable {
-
-  val todoRepository: TodoRepository.Service[Any]
-
-}
-
 object TodoRepository extends Serializable {
 
-  trait Service[R] extends Serializable {
```

Comment:
I think it would be nicer to move these to TodoRepository directly

Reply:
You mean removing the `Service` all together? But then how to define `type TodoRepository = Has[TodoRepository.Service]`? Or did you mean moving this code into the package object?

Reply:
nevermind, I think it's good as is.